src/Particles/pointsCloudSystem.ts

```diff
@@ -0,0 +1,1049 @@
+import { IndicesArray, FloatArray } from "../types";
+import { Color4, Color3 } from "../Maths/math";
+import { Vector2, Vector3, Vector4, TmpVectors, Matrix } from "../Maths/math.vector";
+import { Logger } from "../Misc/logger";
+import { VertexBuffer } from "../Meshes/buffer";
+import { VertexData } from "../Meshes/mesh.vertexData";
+import { Mesh } from "../Meshes/mesh";
+import { EngineStore } from "../Engines/engineStore";
+import { Scene, IDisposable } from "../scene";
+import { CloudPoint, PointsGroup } from "./cloudPoint";
+import { BoundingInfo } from "../Culling/boundingInfo";
+import { Ray } from "../Culling/ray";
+import { PickingInfo } from "../Collisions/pickingInfo";
+import { StandardMaterial } from "../Materials/standardMaterial";
+import { Texture } from "./../Materials/Textures/texture";
+import { Scalar } from "../Maths/math.scalar";
+
+/** Defines the 4 color options */
+export enum PointColor {
+    /** color value */
+    COLOR = 2,

```

Comment:
We try to get enum in camel case and not upper case :)

Reply:
OK. Originally I was copying the way BABYLON.Mesh.DOUBLESIDE was done.

Reply:
Done